src/modules/personenkontext/api/personenkontext.uc.ts

```diff
@@ -183,4 +190,35 @@ export class PersonenkontextUc {
             return SchulConnexErrorMapper.mapDomainErrorToSchulConnexError(result.error);
         }
     }
+
+    public async findRollen(params: FindPersonenkontextRollenBodyParams): Promise<FindRollenResponse> {
+        const anlage: PersonenkontextAnlage = PersonenkontextAnlage.construct(
+            this.rolleRepo,
+            this.organisationRepo,
+            this.dBiamPersonenkontextRepo,
+        );
+        const rollen: Rolle<true>[] = await anlage.findRollen(params.rolleName);
+        const response: FindRollenResponse = {
+            moeglicheRollen: rollen,
+            total: rollen.length,
+        };
+        return response;
+    }
+
+    public async findSchulstrukturknoten(
+        params: FindPersonenkontextSchulstrukturknotenBodyParams,
+    ): Promise<FindSchulstrukturknotenResponse> {
+        const anlage: PersonenkontextAnlage = PersonenkontextAnlage.construct(
+            this.rolleRepo,
+            this.organisationRepo,
+            this.dBiamPersonenkontextRepo,
+        );
+        const ssks: OrganisationDo<true>[] = await anlage.findSchulstrukturknoten(params.rolleId, params.sskName);
+        const sskResponses: OrganisationResponse[] = this.mapper.mapArray(ssks, OrganisationDo, OrganisationResponse);
+        const response: FindSchulstrukturknotenResponse = {
+            moeglicheSkks: sskResponses,
+            total: ssks.length,
+        };
+        return response;
+    }
```

Comment:
Shouldn't this be a part of the controller/endpoints? Since we want to get away from UCs

Reply:
Yes, after the recent discussion about architecture, this should be moved into the controller